installer/bare_cluster.go

```diff
@@ -0,0 +1,73 @@
+package installer
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/golang.org/x/crypto/ssh"
+)
+
+type BareCluster struct {
+	SSHConfig *ssh.ClientConfig
+	Base      *BaseCluster
+
+	startScript string
+}
+
+func (c *BareCluster) InstallFlynn() error {
+	c.Base.SendLog("Installing flynn")
+
+	startScript, discoveryToken, err := c.Base.genStartScript(c.Base.NumInstances)
```

Comment:
Declare `err` as `var err error` and you can assign directly into the struct here.